src/styles/contacts.styl

```diff
@@ -1,9 +1,8 @@
 @require 'settings/palette.styl'
 @require 'components/table.styl'
 
-main > [role=main]
-    width 100%
-    max-width 80rem
+main
+    max-width 75rem
```

Comment:
I change max-width to match with large-screen. So when main touch the border, there is margin around topbar.